Integrations/integration-Lastline.yml

```
@@ -0,0 +1,293 @@
+
+    }
+    var analysisGet = function(uuid,timeout,interval) {
+        if (!timeout) {
+            timeout = lastlineDefaults.timeout;
+        }
+        if (!interval) {
+            interval = lastlineDefaults.interval;
+        }
+        if (!uuid || uuid == '') {
+            throw 'No task UUID provided';
+        }
+        var ec = {};
+        var md = '';
+
+        res = analysisPoll(uuid,timeout,interval);
+
+        if (analysisReady(res.obj.data)) {
+            if (res.obj.data.score > params.malicious_score) {
+                ec.URLMalicious={'url':url,'score':res.obj.data.score};
+            }
+            md+=dataToMd(res.obj.data);
+        } else {
+            md += 'Report not ready.\n';
+            md += 'Task UUID: ' + uuid + '\n';
+        }
+        return {Type: entryTypes.note, Contents: res.obj, ContentsFormat: formats.json, HumanReadable: md, EntryContext: ec};
+    };
+
+    // The command input arg holds the command sent from the user.
+    switch (command) {
+    // This is the call made when pressing the integration test button.
+    case 'test-module':
+        if (!parseInt(params.malicious_score) || parseInt(params.malicious_score) < 0 || parseInt(params.malicious_score > 100)) {
+            throw 'Malicious score threshold invalid. Allowed values are 0-100.';
+        }
+        if (!parseInt(params.full_report_score) || parseInt(params.full_report_score) < 0 || parseInt(params.full_report_score > 100)) {
+            throw 'Full report score threshold invalid. Allowed values are 0-100.';
+        }
+        submitUrl('google.com',0,10);
+        return 'ok';
+    case 'lastlineGet':
+        return analysisGet(args.uuid, args.timeout, args.interval);
+    case 'lastlineUrl':
+        return submitUrl(args.url,args.timeout, args.interval);
+    // case 'lastlineFile':
+    //     return submitFile(args.file_id);
+    default:
```

Remove comment and maybe add throw of unknown command